.envrc.template

```diff
@@ -17,7 +17,6 @@ export API_V3_URL=https://api-dev-green.mbtace.com
 #export CHELSEA_BRIDGE_URL=
 #export CHELSEA_BRIDGE_AUTH=
 
-# URLs of the enhanced trip-update and vehicle-position feeds. Default to the real feed URLs if
-# not set here.
-#export TRIP_UPDATE_URL=
-#export VEHICLE_POSITIONS_URL=
+# URLs of the enhanced trip-update and vehicle-position feeds. Required.
```

Comment:
This says "Required" but it looks like there are defaults provided in `config/runtime.exs`. Should we remove "Required" here or maybe `raise` in `config/runtime.exs` if these values are in the environment?